.drone.star

```diff
@@ -266,6 +267,23 @@ def coverage():
     ]
   }
 
+def changeLog():
+  return {
+    "kind": "pipeline",
+    "type": "docker",
+    "name": "changelog",
+    "image": "registry.cern.ch/docker.io/library/golang:1.16",
```

Comment:
IMO this needs a `"steps": [` section, which will need a "name" and the "image", "commands" and "environment".